pkg/server/sandbox_run.go

```diff
@@ -120,6 +120,11 @@ func (c *criContainerdService) RunPodSandbox(ctx context.Context, r *runtime.Run
 				}
 			}
 		}()
+		ip, err := c.netPlugin.GetPodNetworkStatus(podNetwork)
+		if err != nil {
+			return nil, fmt.Errorf("failed to get network status for sandbox %q: %v", id, err)
+		}
+		sandbox.IP = ip
```

Comment:
Please add comment to explain that some VM based solution is relying on this behavior, and link the issue. And we should try to eliminate the assumption on cri shim implementation detail in the future.